### ISSUE TITLE
Is it possible to add package repositories?

### ISSUE THREAD

**Issue by DrewBarratt** [EnzoAndree/ColabBEAST#3]
Hi Enzo, 

I was just wondering whether it would be possible to add an additional package repository? or to just add an addtional package to the current package folder? Specifically,  I was wodering whether we could import skylinetools from Louis du Plessis repo to make epoch model preperation a little esasier? Let me knwo what you think? https://laduplessis.github.io/skylinetools/package.xml

**Comment by Wytamma** [comment]
Yes, you can use the `Install BEAST2 modules` cell to add packages
